Core/clim-core/dialog-views.lisp

```diff
@@ -101,6 +101,85 @@
                                     :value default
                                     :value-changed-callback #'value-changed-callback))))
 
+;;; radio-box-view
```

Comment:
There is a typo in the first line of the commit message.

Reply:
Ok, I fixed it